### PR TITLE
fix(tag-add): Improved styling of button textcontent

### PR DESCRIPTION
**Pull request by rowa-audil**
BEFORE:
Screenshot 2020-01-14 at 11 21 42

AFTER: 
Screenshot 2020-01-14 at 11 19 36

### CODE REVIEW

File: components/tag/src/tag-add/tag-add.html [hunk 1]
```diff
@@ -6,7 +6,8 @@
     (click)="open()"
     #origin="cdkOverlayOrigin"
   >
-    <dt-icon class="dt-tag-add-icon-button" name="plus-add"></dt-icon> Add Tag
+    <dt-icon class="dt-tag-add-icon-button" name="plus-add"></dt-icon>
+    <div class="dt-tag-add-button-text">Add Tag</div>
```
Comment (tomheller) [inline components/tag/src/tag-add/tag-add.html]: please change to an inline-element like a `span`

File: components/tag/src/tag-add/tag-add.scss [hunk 2]
```diff
@@ -6,37 +6,50 @@
...
   cursor: pointer;
-}
 
-.dt-tag-add-container {
-  position: relative;
-  display: inline-block;
-}
+  .dt-tag-add-container {
+    position: relative;
+    // display: inline-block;
```
Comment (tomheller) [inline components/tag/src/tag-add/tag-add.scss]: remove commented code.

File: components/tag/src/tag-add/tag-add.scss [hunk 3]
```diff
@@ -6,37 +6,49 @@
...
+    vertical-align: middle;
+    line-height: 16px;
+    margin-left: 4px;
   }
 }
 
 .dt-tag-add-overlay {
   width: 320px;
-  height: 82px; // UX requested value: 78
+  height: 80px; // UX requested value: 78
```
Comment (lukasholzer) [inline components/tag/src/tag-add/tag-add.scss]: This comment seems outdated please remove.

File: components/tag/src/tag-add/tag-add.scss [hunk 4]
```diff
@@ -62,12 +74,12 @@
   width: 16px;
   height: 16px;
   vertical-align: text-top;
-  display: inline-block;
   fill: #ffffff;
 }
 
 .dt-tag-add-icon-button {
   fill: $turquoise-600;
+  padding-top: -2px;
```
Comment (lukasholzer) [inline components/tag/src/tag-add/tag-add.scss]: minus padding does not work. This is an invalid value. @tomheller can we add a stylelint rule for such values?

**Comment by lukasholzer** [comment]
Please rebase and squash with a force push only after the review is finished!

This makes reviewing a lot harder!

**Pull request merged**